public/index.html

```diff
@@ -30,7 +30,7 @@
             const botConnection = new BotChat.DirectLine({
                 secret: params['s'],
                 token: params['t']
-                }, params["domain"]
+                }, params["domain"] //the domain should be the DL url https://directline.botframework.com/v3/directline
```

Comment:
if you look at the definition for DirectLine you will see that `domain` is optional. But you do need to pass a secret or a token as the first parameter.